packages/dashboard-frontend/src/containers/FactoryLoader.tsx

```diff
@@ -319,6 +322,14 @@ export class FactoryLoaderContainer extends React.PureComponent<Props, State> {
           return workspace.ref.attributes && workspace.ref.attributes.stackName === attrs.stackName;
         }
         else {
+          // in the case with factory loader an attribute 'stackName' is the factory URL
+          const factoryUrl = attrs.stackName;
```

Comment:
It confuses me a bit. If possible let's not use workspacev1 specific stuff for devworkspaces.

Reply:
I have fixed it.